pyproject.toml

```diff
@@ -445,7 +445,15 @@ fixture-parentheses = false
 # * Disable `flaky` plugin for pytest. This plugin conflicts with `rerunfailures` because provide same marker.
 # * Disable `nose` builtin plugin for pytest. This feature deprecated in 7.2 and will be removed in pytest>=8
 # * And we focus on use native pytest capabilities rather than adopt another frameworks.
-addopts = "-rasl --verbosity=2 -p no:flaky -p no:nose --asyncio-mode=strict"
+addopts = [
+    "-rasl",
+    "--verbosity=2",
+    "-p", "no:flaky",
+    "-p", "no:nose",
+    "-p", "no:legacypath",
```

Comment:
Well, I thought it doesn't add by a mistake, however seems like our tests use legacy LocalPath in some places

```console 
   @pytest.fixture(autouse=True)
    def refuse_to_run_test_from_wrongly_named_files(request):
>       dirname: str = request.node.fspath.dirname
E       AttributeError: 'Function' object has no attribute 'fspath'
```